packages/opentelemetry-api/README.md

```diff
@@ -9,12 +9,112 @@ This package provides everything needed to interact with the OpenTelemetry API,
+
+To collect traces and metrics, you will have to tell the SDK where to export telemetry data to. This example uses Jaeger and Prometheus, but exporters exist for other tracing backends. If you're not sure if there is an exporter for your tracing backend, contact your tracing provider.
+
+#### Tracing
 
-API entry points are defined as global singleton objects `trace` and `metrics` which contain methods used to initialize SDK implementations and acquire resources from the API.
+```javascript
+const { NodeTracerProvider } = require("@opentelemetry/node");
+const { SimpleSpanProcessor } = require("@opentelemetry/tracing");
+const { JaegerExporter } = require("@opentelemetry/exporter-jaeger");
+
+const tracerProvider = new NodeTracerProvider();
+
+/**
+ * The SimpleSpanProcessor does no batching and exports spans
+ * immediately when they end. For most production use cases,
+ * OpenTelemetry recommends use of the BatchSpanProcessor.
+ */
+tracerProvider.addSpanProcessor(
+  new SimpleSpanProcessor(
+    new JaegerExporter(
+      /* export options */
+    )
+  )
+);
+
+/** 
+ * Registering the provider with the API allows it to be discovered
+ * and used by instrumentation libraries. The OpenTelemetry API provides
+ * methods to set global SDK implemetations, but the default SDK provides
+ * a convenience method named `register` which registers sane defaults
+ * for you.
+ * 
+ * By default the NodeTracerProvider uses Trace Context for propagation
+ * and AsyncHooksScopeManager for context management. To learn about
+ * customizing this behavior, see API Registration Options below.
+ */
+tracerProvider.register();
+```
+
+#### Metrics
+
+```javascript
+const api = require("@opentelemetry/api");
+const { MeterProvider } = require("@opentelemetry/metrics");
+const { PrometheusExporter } = require("@opentelemetry/exporter-prometheus");
+
+const meterProvider = new MeterProvider({
+  // The Prometheus exporter runs an HTTP server which
+  // the Prometheus backend scrapes to collect metrics.
+  exporter: new PrometheusExporter({ startServer: true }),
+  interval: 1000,
+});
+
+/** 
+ * Registering the provider with the API allows it to be discovered
+ * and used by instrumentation libraries.
+ * 
+ * Note that if you haven't registered a TracerProvider, you will
+ * be responsible for configuring a context manager and registering
+ * it using the context API.
+ */
+api.metrics.setGlobalMeterProvider(meterProvider);
+```
+
+## Advanced Use
+### API Registration Options
+
+If you prefer to choose your own propagator or context manager, you may pass an options object into the `tracerProvider.register()` method. Omitted or `undefined` options will be replaced by a default value and `null` values will be skipped.
+
+```javascript
+const { B3Format } = require("@opentelemetry/core");
```

Comment:
B3Format  => B3Propagator

Reply:
The rename was not merged when I did this